### PR TITLE
stubs for metering

### PR DESCRIPTION
**Pull request by lamek**
These are the stubs for the upcoming 4.2 metering docs.

I had a messy PR originally, so I copied all the files locally, cut a new branch off master, and copies all the files back up.

### CODE REVIEW

File: modules/metering-install-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Metering Operator.
+
+. Click the Metering card to open its package description, then click Install.
+
+. In the Create Operator Subscription screen, select the metering namespace you
+created above in the *A specific namespace on the cluster* drop-down, and
+specify your update channel and approval strategy. Click *Subscribe* to install
+the metering-operator into your selected namespace.
+
+. On the *Subscription Overview* screen, the *Upgrade status* indicates 1
```
Comment (huffmanca) [inline modules/metering-install-operator.adoc]: This is a personal preference, but I would reword this to be a bit clearer about the `1 installed`. What about:

> ...the *Upgrade status* indicates when the Metering Operator has finished installing. Once installed, this count will be `1`.

File: modules/metering-install-operator.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+installed when the Metering Operator has finished installing. Click the 1
+installed (or installed version) link to view the ClusterServiceVersion overview
```
Comment (huffmanca) [inline modules/metering-install-operator.adoc]: Reword this sentence to remove parenthesis.

**Review by huffmanca — COMMENTED** [review]
A general comment on most of your modules - make the `id` and header line separate. Currently these are one line. While it should render correctly, it's typically easier for humans to have the header line beginning immediately beneath it.

Also, I'm hesitant on merging a large number of stubs into a repo. Right now these seem used for your own personal tracking, but they're not ready for public consumption. I'd rather we merge the various pieces as they're needed. For instance, each assembly could be its own PR that is reviewed individually; however, you could keep this structure intact for your use to build out the assemblies.

Let me know your thoughts on this.

File: metering/metering-installing-metering.adoc [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
 = Installing Metering
-include::modules/common-attributes.adoc[]
+include::modules/common-attributes.adoc[] 
 :context: metering 
 
 toc::[]
 
-Installing metering stub
+On OpenShift 4.x clusters the Metering Operator is available through Operator
+Hub and can be installed using the Operator Lifecycle Manager (OLM). If you're
```
[the remark this answers is not in the document]
Comment (lamek) [reply metering/metering-installing-metering.adoc]: That sounds good. Per our discussion elsewhere I'm going to scrap this PR. We'll make some smaller ones per assembly.

**Comment by lamek** [comment]
Closing this. I'm going to make a set of smaller PRs, one per section, with fully completed stubs.
